.gitignore

```diff
@@ -0,0 +1,39 @@
+target/
```

Comment:
For the future: It would be better not to add general changes like adding .gitignore and pom.xml to the same commit with any meaningful code changes. 
In your case, it would be better to split changes to some initial commit which is setting up the project and separate commit for homework#3
Just a small comment, no need to do something with it right now